packages/cli/src/commands/bundle/assetCatalogIOS.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import path from 'path';
+import fs from 'fs-extra';
+import assetPathUtils from './assetPathUtils';
+import {AssetData} from './buildBundle';
+
+export async function cleanAssetCatalog(catalogDir: string): Promise<void> {
+  await Promise.all(
+    (await fs.readdir(catalogDir))
```

Comment:
When there's no other async actions going on in the FS, it's generally faster to use sync alternatives. That's one of the reasons we generally use sync FS access in CLIs (when there's nothing else to block), but async on the servers (where we could block requests).